packages/modal/index.js

```diff
@@ -0,0 +1,23 @@
+import { ModalTitleClose } from './modal-title-close.js'
+import { ModalTitleBar } from './modal-title-bar.js'
+import { ModalFooter } from './modal-footer.js'
+import { ModalMain } from './modal-main.js'
+import { CanCloseMixin, ProvidesCanCloseToSlotsMixin } from './util.js'
+
+if (!customElements.get('w-modal')) {
+  customElements.define('w-modal', ModalMain)
+  customElements.define('w-modal-title-bar', ModalTitleBar)
+  customElements.define('w-modal-title-close', ModalTitleClose)
+  customElements.define('w-modal-footer', ModalFooter)
+}
+
+export {
+  ModalMain as WarpModal,
```

Comment:
Maybe there needs to be only 1 export here rather than all these, since you just need to import the file and you'll get all the custom elements thru the `.define` rather than these exports

Reply:
Again just following convention from other components 🤷